assets/sass/main.scss

```diff
@@ -0,0 +1,101 @@
+@import "reset";
+@import "variables";
+
+body {
+  background: #dddd url(../images/background.jpg) no-repeat ;
+  background-size: cover;
+}
+
+.slider {
+  max-width: $slider-width;
+  height: $slider-height;
+  margin: 25px auto;
+  position: relative;
+
+  &-dots {
```

Comment:
Những cái slector to như này, ví dụ như &-dots, &-main, ... nên viết tách thành cái mới nhé. Viết thế này sẽ gây khó hiểu, mất công cuộn lên để xem "&" đang là gì